commands.js

```diff
@@ -790,7 +790,7 @@ exports.commands = {
 		let rankLists = {};
 		for (let u in targetRoom.auth) {
 			if (!rankLists[targetRoom.auth[u]]) rankLists[targetRoom.auth[u]] = [];
-			rankLists[targetRoom.auth[u]].push(u);
+			rankLists[targetRoom.auth[u]].push((u in targetRoom.users ? "**" + u + "**" : u));
```

Comment:
The extra brackets don't aid readability at all. On the other hand, if you wanted, you could put

`.push((u in targetRoom.users) ? "**" + u + "**" : u)`

But that's not really necessary, either.